docs/src/main/asciidoc/project-features.adoc

```diff
@@ -54,7 +54,7 @@ For configuration options please check the `spring.sleuth.otel.exporter.jaeger`
 [[features-otel-otlp]]
 ==== OpenTelemetry OTLP Integration
 
-We're providing an out-of-the-box integration with the OTel Jaeger exporter. via the `io.opentelemetry:opentelemetry-exporter-otlp` dependency.
+We're providing an out-of-the-box integration with the OTel gRPC exporter. via the `io.opentelemetry:opentelemetry-exporter-otlp` dependency.
```

Comment:
I think this should actually say "OTel OTLP gRPC exporter", since there is also a grpc-based Jaeger exporter available.

Reply:
At the beginning, i named it like that but after thinking about it "OTel (OpenTelemetry) OTLP (OpenTelemetry) gRPC exporter" seems to be redundant.

And in the doc, the chapitre before is about Otel Jaeger exporter.

Reply:
Ok, the title is __OpenTelemetry OTLP Integration__ .
So, I push a new commit.